kuryr_kubernetes/controller/managers/prometheus_exporter.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2020 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import netaddr
+
+from flask import Flask
+from flask import Response
+
+from oslo_config import cfg
+from oslo_log import log as logging
+
+from kuryr.lib._i18n import _
+from kuryr_kubernetes import clients
+from kuryr_kubernetes import config
+
+from prometheus_client import CollectorRegistry
```

Comment:
That should be along with flask. Also normally in OpenStack we don't import objects directly.